### PR TITLE
Sizeresponsible grid within grid

### PR DESCRIPTION
**Pull request by anthonykoerber**
@jcmoore @dlasky @shuwen This addresses the immediate recycler-within-recycler gotchas. Merge-able, but since this touches critical infrastructure, figured I'd PR.

### CODE REVIEW

**Comment by shuwen** [comment]
looks like you need to update the tests as well 😜

File: src/strand-item-recycler/strand-item-recycler.js [hunk 1]
```diff
@@ -548,9 +553,11 @@ found here: https://github.com/Polymer/core-list
 		},
 
 		_needsInitialization: function (data, _templateFound) {
-			this._initializable = false;
-			this._initialized = false;
-			this.initialize();
+			if (this._isAttached) {
```
Comment (dlasky) [inline src/strand-item-recycler/strand-item-recycler.js]: unclear about the flow internally here, but what happens in the case where it is attached some time later?

File: src/shared/behaviors/sizeresponsible.html [hunk 2]
```diff
@@ -28,15 +28,19 @@
 
 			var div = document.createElement("div");
 			div.style.cssText = "position:absolute; width:auto; height:auto; top:0; right:0; bottom:0; left:0; margin:0; padding:0; overflow:hidden; visibility:hidden; z-index:-1";
+
 			var iframe = document.createElement("iframe");
+			iframe.cb = fn;
```
Comment (dlasky) [inline src/shared/behaviors/sizeresponsible.html]: what is this for?

**Comment by dlasky** [comment]
so as this stands now we will effectively be leaking iframes if you add the listener to teh same target more than once.

We should probably add some really basic state to just not init the iframes again if its already been done.

I'm not sure if you guys think we should destroy the iframes as well when removed?

**Comment by anthonykoerber** [comment]
The failing test lead me to take another look at this in FF - the performance is "less than optimal". I think we need to spend some more time looking at the solution. So far, I've been unable to positively identify the source of the problem. @jcmoore @dlasky

**Pull request closed without merging**